rkt/metadata_service.go

```diff
@@ -335,20 +340,25 @@ func handlePodAnnotation(w http.ResponseWriter, r *http.Request, pm *schema.PodM
 		return
 	}
 
-	w.Header().Add("Content-Type", "application/json")
+	w.Header().Add("Content-Type", "text/plain; charset=us-ascii")
```

Comment:
Annotations get marshaled/unmarshaled to/from JSON, hence I'd imply utf-8 encoding.